apps/preview/next/pages/showcases/Footer/FooterBasic.tsx

```diff
@@ -0,0 +1,265 @@
+      },
+    ],
+  },
+  {
+    label: 'Help',
+    subcategories: [
+      {
+        subcategoryLabel: 'Help centers',
+        link: '/help',
+      },
+      {
+        subcategoryLabel: 'Security & fraud',
+        link: '/security',
+      },
+      {
+        subcategoryLabel: 'Feedback',
+        link: '/feedback',
+      },
+      {
+        subcategoryLabel: 'Contact',
+        link: '/contact',
+      },
+    ],
+  },
+  {
+    label: 'Services',
+    subcategories: [
+      {
+        subcategoryLabel: 'Gift cards',
+        link: '/gift',
+      },
+      {
+        subcategoryLabel: 'Order pickup',
+        link: '/order',
+      },
+      {
+        subcategoryLabel: 'Purchase status',
+        link: '/purchase',
+      },
+      {
+        subcategoryLabel: 'Track orders',
+        link: '/track',
+      },
+    ],
+  },
+  {
+    label: 'About',
+    subcategories: [
+      {
+        subcategoryLabel: 'About us',
+        link: '/about',
+      },
+      {
+        subcategoryLabel: 'Order pickup',
+        link: '/order',
+      },
+      {
+        subcategoryLabel: 'Purchase status',
+        link: '/purchase',
+      },
+      {
+        subcategoryLabel: 'Track orders',
+        link: '/track',
+      },
+      {
+        subcategoryLabel: 'Returns',
+        link: '/returns',
+      },
+    ],
+  },
+];
+const socialMedia = [
+  {
+    label: 'Facebook',
+    link: '/facebook',
+    icon: 'SfIconFacebook',
+  },
+  {
+    label: 'Twitter',
+    link: '/twitter',
+    icon: 'SfIconTwitter',
+  },
+  {
+    label: 'Instagram',
+    link: '/instagram',
+    icon: 'SfIconInstagram',
+  },
+  {
+    label: 'Pinterest',
+    link: '/pinterest',
+    icon: 'SfIconPinterest',
+  },
+  {
+    label: 'Youtube',
+    link: '/youtube',
+    icon: 'SfIconYoutube',
+  },
+];
+const contactOptions = [
+  {
+    label: 'Help center',
+    link: '/help',
+    details: ['Find answers online anytime'],
+    icon: 'SfIconHelp',
+  },
+  {
+    label: 'Live chat',
+    link: '/live',
+    details: ['Mon–Fri, 5am–10pm PT', 'Sat–Sun, 6am–9pm PT'],
+    icon: 'SfIconContactSupport',
+  },
+  {
+    label: '1 234 567 8901',
+    link: '/tel',
+    details: ['Mon–Fri, 5am–10pm PT', 'Sat–Sun, 6am–9pm PT'],
+    icon: 'SfIconCall',
+  },
+];
+const bottomLinks = [
+  {
+    label: 'Terms',
+    link: '/terms',
+  },
+  {
+    label: 'Privacy policy',
+    link: '/privacy',
+  },
+];
+
+const iconsMap: {
+  [key: string]: any;
+} = {
+  SfIconFacebook,
+  SfIconTwitter,
+  SfIconInstagram,
+  SfIconPinterest,
+  SfIconYoutube,
+  SfIconHelp,
+  SfIconContactSupport,
+  SfIconCall,
+};
+export default function FooterBasic() {
+  return (
+    <div className="pt-10 bg-neutral-100">
+      <div className="grid justify-center grid-cols-[1fr_1fr] md:grid-cols-[repeat(4,1fr)] px-4 md:px-6 pb-10 max-w-[1536px] mx-auto">
+        {categories.map(({ label, subcategories }) => (
+          <div className="min-w-[25%] xs:min-w-[50%] xs:pb-4" key={label}>
```

Comment:
question: not a big difference, but I think it might be easier to just use grid and grid-cols here?

Reply:
Changed it thx.